example/test/unmount_test.dart

```diff
@@ -19,7 +18,6 @@ class SimpleComponent extends react.Component {
 
 void main() {
   print("What");
-  setClientConfiguration();
```

Comment:
Omg being able to remove all of these is so nice!!

Reply:
💯And removing the react_client import altogether feels awesome haha